source/_integrations/denonavr.markdown

```diff
@@ -38,6 +40,8 @@ Known supported devices:
 - Other Denon AVR receivers (untested)
 - Marantz receivers (experimental)
 
+If your model is not on the list then give it a test, if everything works correctly then add it to the list on [GitHub](https://github.com/home-assistant/home-assistant.github.io/tree/current/source/_integrations/denonavr.markdown).
```

Comment:
I think that it's better to point people to the **edit on Github** button

Reply:
Sounds good. I have updated the language accordingly.